src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -133,30 +130,43 @@ Here are two query bar examples:
   </Collapser>
 </CollapserGroup>
 
+### Trace groups [#trace-groups]
+
+The default view of distributed tracing shows traces grouped by the same root entry span. In other words, traces are grouped by the span where New Relic began recording the request. You can slide the toggle **Group similar traces** to turn this on and off.
+
+With trace groups you get a high-level view of traces so you can understand request behavior for groups of similar traces. This is helps you understand dips or spikes in trace count, duration, and errors. 
```

Comment:
Remove "is" after "This" in the second sentence!

Reply:
@urbiz-nr, good catch!